src/components/List/PureRowContent.js

```diff
@@ -0,0 +1,229 @@
+import React, { Component } from "react";
+import styled from "styled-components";
+import PropTypes from "prop-types";
+import classnames from "classnames";
+
+import spacing from "../../theme/spacing";
+import colors from "../../theme/colors";
+import colorThemes from "../../theme/colorThemes";
+
+import { Row, Column } from "../Grid";
+import OverflowIcon from "../Icons/Overflow";
+import { mediumAndUp } from "../../theme/mediaQueries";
+
+import RowToggler, { IconButton } from "./RowToggler";
+import constants from "../../theme/constants";
+
+const RowWrapper = styled.div`
+  background-color: ${colors.white.base};
+  border-bottom: 1px solid ${colorThemes.global.gray04};
+  border-top: 1px solid ${colorThemes.global.gray04};
+  border-right: 1px solid ${colors.white};
+  border-left: 1px solid ${colors.white};
+  margin-bottom: -1px;
+  padding: 0 ${spacing.cozy};
+  &:first-child {
+    border-top: 0px;
+  }
+
+  ${mediumAndUp`
+    padding: 0 ${spacing.normal};
+    &.row__wrapper--expanded {
+      transition: box-shadow 0.3s ${
+        constants.easing.easeInOutQuad
+      }, margin-bottom 0.3s ${constants.easing.easeInOutQuad};
+      margin-bottom: 3px;
+      margin-top: 3px;
+      border-radius: 4px;
+      box-shadow: 0 4px 4px 0 rgba(0, 0, 0, 0.06), 0 0 4px 0 rgba(0, 0, 0, 0.12);
+      border: 1px solid ${colorThemes.global.gray04};
+      overflow: hidden;
+    }
+    
+    &.row__wrapper--collapsed {
+      transition: box-shadow 0.3s ${
+        constants.easing.easeInQuad
+      }, margin-bottom 0.3s ${constants.easing.easeInQuad};
+      box-shadow: 0;
+      margin-bottom: -1px;
+    }
+  `};
+
+  .button--expand-or-collapse {
+    position: absolute;
+    right: ${props => (props.rowTriggerPosition === "right" ? "0" : "auto")};
+    left: ${props => (props.rowTriggerPosition === "left" ? "0" : "auto")};
+    z-index: 10;
+    margin: 0;
+    padding: 0;
+    /* spacing.spacious is spacing.normal * 2 - paddings for ListContainer */
+    height: calc(100% - ${spacing.spacious});
+  }
+`;
+
+const ListContainer = styled.div`
+  background-color: ${colors.white.base};
+  align-items: stretch;
+  display: flex;
+  padding-top: ${spacing.normal};
+  padding-bottom: ${spacing.normal};
+  position: relative;
+`;
+
+const OverflowDesktopContainer = styled(Column)`
```

Comment:
This looks very similar to the `OverflowDesktopContainer` component in `/src/components/List/RowContent`. Can you please import and extend that component, rather than duplicating it?

Reply:
Yep, done.